.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Release Charts and publish docs
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish-docs:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
Can you add a comment as to where this email comes from?

Reply:
I have no idea either lol. I think I blindly copy pasted it from mkdocs example. I fixed it and it's now:
```
          git config user.name "$GITHUB_ACTOR"
          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```
which is the same as the helm release action